controllers/configurationpolicy_controller.go

```diff
@@ -1045,277 +1225,250 @@ func (r *ConfigurationPolicyReconciler) handleDeletion(plc *policyv1.Configurati
 	return nil
 }
 
-// handleTemplatization sets the `plc.Spec.ObjectTemplates` after resolving any templatized values.
-func (r *ConfigurationPolicyReconciler) handleTemplatization(
-	plc *policyv1.ConfigurationPolicy, usingWatch bool,
-) (parentStatusUpdateNeeded bool, err error) {
-	log := log.WithValues("policy", plc.GetName())
-
-	if r.EnableMetrics {
-		startTime := time.Now().UTC()
+type minimumMetadata struct {
```

Comment:
Oh, nifty. I don't think I had ever considered marshalling directly to something smaller than the full object. 🙂

Reply:
I'll admit, I saw something similar to parse the GVK from an object in a Kubernetes library, but it didn't include the name and namespace.